students/daniel_carrasco/session03/mailroom.py

```diff
@@ -1,35 +1,52 @@
 #!/usr/bin/env python3
 def main():
-
-    print('\noptions\n')
-    print('A = Send a Thank You')
-    print('\n')
-
-    response = input('what do you want to do?').upper()
-    if response == 'A':
+    print('\nChoose an action(1 - 3):\n\n\
+            1 - Send a Thank You to a single donor.\n\
+            2 - Create a Report.\n\
+            3 - Quit\n')
+    choice = int(input(''))
+    if choice == 1:
         thankyou()
-    pass
+    elif choice == 2:
+        report()
+    elif choice == 3:
+        quit()
+    else:
+        main()
```

Comment:
calling main() inside of main won't really work well.  Also for choice 3, i'd suggest just doing sys.exit()